### ISSUE TITLE
warning: kqueue is not supported on this platform

### ISSUE THREAD

**Issue by auxbuss** [tweetstream/tweetstream#181]
tweetstream-2.6.1/lib/tweetstream/client.rb:401: warning: kqueue is not supported on this platform

Just started getting this warning message in the logs, which I think has been triggered by updating eventmachine from 1.0.7 to 1.0.8. This corresponds to line 418 in the current source.

    EventMachine.kqueue

I presume this is okay -- being only a warning, and all seemingly working okay -- but could you confirm.

Also, is there a way to stop the message?

Thanks.

**Comment by stve** [comment]
My recollection is that these no-op but i'll check.

**Comment by stve** [comment]
Yep, it's just a warning: https://github.com/eventmachine/eventmachine/blob/67fd2af021159fc8a8b72f24bd6eff6f45669bdb/ext/rubymain.cpp#L1040-L1049

I think I can suppress the message by changing tweetstream's code from `EM.kqueue` to `EM.kqueue = true if EM.kqueue?` which is actually how they use it in their tests: https://github.com/eventmachine/eventmachine/blob/36003fcf1675e8718a9d45b83e70c2054ddb7c40/tests/test_file_watch.rb#L28-L34

It does appear that this was changed recently: https://github.com/eventmachine/eventmachine/commit/9af94064e1f05b373145ba0c3cac41daa9daf369

**Comment by stve** [comment]
heh, it could probably be simplified to `EM.kqueue = EM.kqueue?` which seems silly but ¯\_(ツ)_/¯

**Comment by auxbuss** [comment]
Cool. Thanks.
